src/pages/todos/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useState } from 'react';
+
+import { StyledTodoForm, StyledTodoInput, StyledTodosContainer } from '@pages/todos/Todos.styles';
```

Comment:
you should define an `index` file inside directories like `pages` and import everything from there. This way the other files do not care if the internal folder structure of these directories are changed.